cartographer/ground_truth/compute_relations_metrics_main.cc

```diff
@@ -126,7 +127,7 @@ void WriteRelationMetricsToFile(const std::vector<Error>& errors,
          "expected_translation_x,expected_translation_y,expected_"
          "translation_z,expected_rotation_w,expected_rotation_x,"
          "expected_rotation_y,expected_rotation_z,covered_distance\n";
-  for (size_t relation_index = 0; relation_index < ground_truth.relation_size();
+  for (int relation_index = 0; relation_index < ground_truth.relation_size();
```

Comment:
Why int?

Reply:
compiler warning about comparison between signed and unsigned. `relation_size()` is int.